internal/envtest/envtest.go

```diff
@@ -84,15 +102,14 @@ func (e *Environment) start() {
 
 	//+kubebuilder:scaffold:scheme
 
-	k8sClient, err := client.New(cfg, client.Options{Scheme: scheme.Scheme})
-	if err != nil {
-		panic(err)
-	}
-
-	e.intClient = k8sClient
+	e.Client = k8sClient
+	// e.Kubeconfig, err = CreateKubeconfigFileForRestConfig(e.env.Config)
+	// if err != nil {
+	// 	panic(err)
+	// }
 }
 
-func (e *Environment) stop() {
+func (e *Environment) Stop() {
```

Comment:
question: Is there a reason to export Stop() ? I can't see it being called outside this package.

Reply:
Can you think of a reason not to export it? It seems that Start and Stop are two fundamental operations of one of these environments. The fact that both are typically only invoked by .Run shouldn't really matter.

I'm reluctant to touch this since it is involved in my later kind backend PR.

Reply:
If there's no reason to export something, don't do it. Keep things in private scope unless there's a good reason to not to. This PR alone provides no good reason to export them (and I don't know if the other PR does either).

Reply:
See my above comment. This one can't rise to the level of an issue since it does not functionally affect things, I'll assume that this is non blocking, and am open to having that discussion on the kind/envtest PR.